app/models/page_sequence.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# Represents the pages in the claim process.
+#
+# Depending on the state of `claim`,  the sequence of slugs will change,
+# filtering out pages that are not relevant. For example, the sequence for a
+# claim that states it is still employed at the same school as the
+# `claim_school` will skip the `current-school` page as `current_school` is
+# inferred to be the same as `claim_school.
+class PageSequence
```

Comment:
Just wondering if this should be called TslrPageSequence, as there will no doubt be a sequence for Maths and Physics too? There may be some logic we can share too

Reply:
The answer to that one is: I don't think we know enough to decide yet. It may be that passing in the claim will give the generic `PageSequence` enough to create a policy-appropriate sequence. There will almost certainly be logic shared between the two, but how that shakes out will be easier to decide at the point we are introducing a second sequence.

Reply:
For example, maybe we end up with a generic `PageSequence` with policy-specific rules/slugs that it is composed of. We'll be better equipped to decide when we're working on the work to introduce Maths and Physics support

Reply:
Yep, cool. Fair enough. I think this can easily be changed further down the line 👍